### ISSUE TITLE
an atom may not have `@` attribute

### ISSUE THREAD

**Issue by yegor256** [objectionary/lints#62]
This should be prohibited:

```
[@] > foo /number
```

Syntactically, it'a s valid EO program, but it must be prohibited to use `@` in atoms. Let's issue a `critical` defect in this case.

**Comment by yegor256** [comment]
@maxonfjvipon am I right?

**Comment by h1alexbel** [comment]
@yegor256 currently with `0.49.2` parser version, I getting this:

```text
[ERROR] Failed to parse '/Users/aliaksei/e/parsed/foo.eo:2': [2:1] error: no viable alternative at input:
[@] > foo /number
 ^
EO sources parsed in .eoc
```

XMIR:

```xml
<program xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         dob="2024-12-20T13:59:52"
         ms="31"
         name="foo"
         revision="b7b0283"
         source="/Users/aliaksei/e/parsed/foo.eo"
         time="2024-12-26T17:51:55.602703Z"
         version="0.49.2"
         xsi:noNamespaceSchemaLocation="https://www.eolang.org/xsd/XMIR-0.49.2.xsd">
   <listing># Foo.
[@] &gt; foo /number
</listing>
   <objects/>
   <errors>
      <error check="eo-parser" line="2" severity="critical">[2:1] error: no viable alternative at input:
[@] &gt; foo /number
 ^</error>
   </errors>
   <sheets>
      <sheet>stars-to-tuples</sheet>
      <sheet>wrap-method-calls</sheet>
      <sheet>const-to-dataized</sheet>
   </sheets>
</program>
```

As far I understand, `φ` in atoms already handled at the parser level, and we can close this issue, correct?

**Comment by yegor256** [comment]
@h1alexbel it's a bug in the parser, I believe: https://github.com/objectionary/eo/issues/3768

**Comment by h1alexbel** [comment]
@yegor256 assign me, please
